core/server/api_container/server/api_container_service.go

```diff
@@ -810,6 +827,8 @@ func convertServiceStatusToServiceInfoStatus(serviceStatus service.ServiceStatus
 		return kurtosis_core_rpc_api_bindings.ServiceStatus_RUNNING, nil
 	case service.ServiceStatus_Stopped:
 		return kurtosis_core_rpc_api_bindings.ServiceStatus_STOPPED, nil
+	case service.ServiceStatus_Registered:
```

Comment:
@laurentluce i had to add support for this as perhaps making the call an individual call is just too fast for the APIC